rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/utils/datasync/BrokerBasedLog.java

```diff
@@ -90,27 +90,24 @@ public BrokerBasedLog(ConnectConfig connectConfig,
 
         this.topicName = topicName;
         this.dataSynchronizerCallback = dataSynchronizerCallback;
-        producer = new DefaultMQProducer();
+        this.producer = new DefaultMQProducer();
         this.producer.setNamesrvAddr(connectConfig.getNamesrvAddr());
+        this.producer.setInstanceName(ConnectUtil.createInstance(connectConfig.getNamesrvAddr()));
         this.producer.setProducerGroup(consumerId);
```

Comment:
It is recommended to use producerId/producerGroup config instead of consumerId.

Reply:
I have not modified this name.It has been called consumerId，I will modify it name is workid